src/higlass_schema/cli.py

```diff
@@ -10,7 +10,7 @@
 
 
 def export(_: argparse.Namespace) -> None:
-    console.print_json(schema_json(indent=2))
+    print(schema_json())
```

Comment:
Any reason to drop indentation entirely?

Reply:
No reason, just thinking that's the job of another tool. I can add it back.

Reply:
I guess you could make it an option to `export`